### ISSUE TITLE
filetea 0.1.16 does not support jquery 2

### ISSUE THREAD

**Issue by dront78** [elima/FileTea#1]
firebug live() is not a function and no file panel
downgrade to jquery 1.8.3 work as expected

**Comment by elima** [comment]
@dront78, yes, I ran into that problem when releasing 0.1.16, and I decided to freeze it in 1.8.3 because I'm already working on FileTea 2 which will change code base dramatically. In fact, I moved jQuery 1.8.3 source code inside FileTea to avoid distros shipping a newer jQuery to break FileTea.

I prefer to devote my (very) limited time to FileTea 2 instead of fixing these issues in current version of the UI, which will no longer be used. I plan to move away from jQuery-UI.

**Comment by dront78** [comment]
nice to hear about move forward
thank you
